R/brmsfit-methods.R

```diff
@@ -391,6 +391,83 @@ VarCorr.brmsfit <- function(x, sigma = 1, summary = TRUE, robust = FALSE,
   lapply(tmp, .VarCorr)
 }
 
+#' Extract Latent Autocorrelated Effects
```

Comment:
This function has very limited application for an exported function. Why do we need it? I would prefer not to have such a function being exported?

Reply:
I mainly added this in response to a question from a colleague who was experimenting with my fork -- they wanted a way to extract the latent variables analogous to calling `ranef` on a fitted model to extract group-level parameters. I'd be happy to do this in a different way if that would fit better.

Reply:
I would prefer to not have this function as part of this PR as it is too specific. I would like to think through separately if there is a good and general enough function and use-case that would contain this here as a special case. Perhaps we can discuss in a separate issue after this PR has been merged (without the function for now).